scripts/exdump_monitor.sh

```diff
@@ -851,7 +851,7 @@ DTIM_latest_crisf4=-0.51
 DTIM_earliest_iasidb=-1.50
 DTIM_latest_iasidb=-0.51
 
-$ushscript_dump/bufr_dump_obs.sh $dumptime 0.50 1 atovs omi esamua \
+$USHbufr_dump/bufr_dump_obs.sh $dumptime 0.50 1 atovs omi esamua \
```

Comment:
Revert to using original variable, $ushscript_dump. It is already defined in JOBSPROC_DUMP_MONITOR.